force-app/main/default/lwc/miscPermissionBasedUI/miscPermissionBasedUI.html

```diff
@@ -10,9 +10,9 @@
         </div>
         <c-view-source source="lwc/miscPermissionBasedUI" slot="footer">
             Use custom permission to conditionally render UI. To change the
-            rendered UI assign the
-            <strong>Access_Restricted_UI</strong> permission set to this user
-            and refresh the page.
+            rendered UI assign
+            the&nbsp;<strong>Access_Restricted_UI</strong>&nbsp;permission set
```

Comment:
A suggestion to avoid hard-coding escape characters would be to use the `prettier-ignore` comment option on the outer `div`, see example implementation [here](https://github.com/trailheadapps/visualforce-to-lwc/blob/master/force-app/main/default/lwc/editRecordWrapper/editRecordWrapper.html#L12).